### ISSUE TITLE
F2P Notice

### ISSUE THREAD

**Issue by Drutol** [Mordonus/RaidOps#416]
CRB has released addon API changelog together with f2p launch. Up to this point I could only guess what has been changed.

Things that I noticed when I logged in for a few minutes : 
- Armory runes are not being collected properly
- Empty GP value is displayed on items that have Imprint string in their names and are not item tokens.

I'll try to resolve those issues today. I may fail to do so as there's university opening ceremony for me to attend. Tomorrow those issues ( and probably more ) will be resolved.

**Comment by Drutol** [comment]
Website item db has to be redone , it may be unavailable for some time. I just need to create 100k entries there :D

**Comment by Drutol** [comment]
Website is now down... removing old item db halfway through

**Comment by Hysteriana** [comment]
omg omg ... so much room for typos...

**Comment by Drutol** [comment]
Populating it with new stuff

**Comment by Drutol** [comment]
Umm no typos this time , everything is generated. I don't have to write anything , aside from commands :D

**Comment by Drutol** [comment]
Tbh , I made one typo in command...

**Comment by Hysteriana** [comment]
damn i knew it 😁

**Comment by Drutol** [comment]
Shhhhh... I was writing at tremendous speed.

**Comment by Drutol** [comment]
Wat... each item = different runeset now?

**Comment by Hysteriana** [comment]
yes and no

**Comment by Drutol** [comment]
I've read a guide on those runes and I don't see any answer there... is it enough to write this for example?

> Striker - Head | 2
> Striker - Chest | 2

**Comment by Hysteriana** [comment]
tl;dr : class set are unique, general set are not. each item counts for it
own set bonus. head 2 + chest 2 = 2x 2set bonus. head 4 + chest 2 = 1x 4set
bonus + 1x 2set bonus

**Comment by Drutol** [comment]
Mhm , I'll have to invent some way to collect it neatly. :)

**Comment by Hysteriana** [comment]
check out the new runemaster addon

**Comment by Drutol** [comment]
Nah , I just need to index every set and count it . I want it to be compatible with current website import.

**Comment by Hysteriana** [comment]
yeah I know. it would just give you an idea what you are dealing with

**Comment by Drutol** [comment]
In terms of code to interact with runes I'm done. It just surprised me that I got double striker.

**Comment by Drutol** [comment]
All known issues are now resolved...
I'll now update armory module appropriately.
After this if there are no requests I'll either:
- Update standalone Master Loot addon.
- Resume work on LootHex.
- Addon to draw on minimap and assign ppl to drawn things. (? idea from guy from reddit ?)

I don't know yet what I'll want to do and whether I'll have time for this... (studies...)
